README.md

```diff
@@ -1,70 +1,71 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Math Magicians
 
-## Available Scripts
+> "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations and read a random math-related quote.
 
-In the project directory, you can run:
 
-### `npm start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- React app
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Live Demo (if available)
 
-### `npm test`
+[Live Demo Link](https://livedemo.com)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-### `npm run build`
+## Getting Started
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+Run the following commands to clone and run it.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### To clone the repository
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+  `git clone git@github.com:ecekpo/math_magician_setup.git`
 
-### `npm run eject`
+### To enter Directory
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+`cd math_magician_setup`
 
```

Comment:
[ OPTIONAL CHANGE ] This change won't stop your project from being approved but will help improve your project documentation. Please consider making the changes marked as optional 👍

- [ ] Kindly add the instruction to install project packages after this line above before the project is started
- [ ] Add `npm install` below this line 👍